scripts/artifactory-cleanup.js

```diff
@@ -0,0 +1,69 @@
+const exec = require('child_process').exec;
+const semver = require('semver');
+const packageJson = require('../package.json');
+
+const BASE_ARTIFATORY_URL = 'https://artifactory.wikia-inc.com/artifactory';
+const BASE_PACKAGE_PATH = '/npm-local/@fandom/jwplayer-fandom/-/@fandom';
+const BASE_READ_PATH = `/api/storage${BASE_PACKAGE_PATH}`;
+const BASE_DELETE_PATH = BASE_PACKAGE_PATH;
+
+const currentMajorVersion = semver.major(packageJson.version);
+const previousMinorVersion = semver.minor(packageJson.version) - 1;
+const versionThreshold = `${currentMajorVersion}.${previousMinorVersion}.0`;
+
+async function getArtifactoryTokenFromYarnConfig() {
+	return new Promise((resolve, reject) => {
+		exec(
+			'yarn config get "//artifactory.wikia-inc.com/artifactory/api/npm/wikia-npm/:_authToken"',
+			(err, stdout, stderr) => {
+				if (stderr) {
+					reject(stderr);
+				} else {
+					resolve(stdout.trim());
+				}
+			},
+		);
+	});
+}
+
+function extractVersionFromPackageName(packageName) {
+	return packageName.match(/^\/jwplayer-fandom-(\d+\.\d+\.\d+).*\.tgz/)?.[1];
+}
+
+function isTestVersion(packageName) {
```

Comment:
test version = every version that is not valid semver (`X.Y.Z`)

Reply:
This comment could be a part of a code documentation 😉 It might be a good idea to copy this detail into a jsdoc.